### PR TITLE
Use hd5 files

### LINKED ISSUE #27: Speed up originality by using hd5 files
hd5 files are a lot faster to read from disk than CSVs. If we change get_submission to write out an hd5 the first time it sees a submission, and then load up an hd5 the next time it is called this will speed up originality when the number of submissions is larger than the cache

### PR DESCRIPTION
**Pull request by cpurta**
Will resolve #27

### CODE REVIEW

**Comment by zoso95** [comment]
This looks good to me. @philipcmonk any objections to this?

File: originality.py [hunk 1]
```diff
@@ -49,8 +49,8 @@ def get_submission(db_manager, filemanager, submission_id):
         logging.getLogger().info("Could not get submission {}".format(submission_id))
         return None
 
-    df = pd.read_csv(local_file)
-
+    df = pd.read_hdf(local_file,'submission_data')
+
```
Comment (philipcmonk) [inline originality.py]: Remove trailing whitespace.

**Comment by cpurta** [comment]
removed the trailing whitespace

**Comment by zoso95** [comment]
If need be, I can checkout master and make a clean commit on this

File: submission_criteria/originality.py [hunk 2]
```diff
@@ -52,10 +52,7 @@ def get_submission(db_manager, filemanager, submission_id):
         logging.getLogger().info("Could not get submission {} at S3 path {}".format(submission_id, s3_filename))
         return None
 
-    df = pd.read_csv(local_file)
-    assert "id" in df.columns, "No id column in submission {}".format(s3_filename)
-    assert "probability" in df.columns, "No probability column in submission {}".format(s3_filename)
```
Comment (xanderdunn) [inline submission_criteria/originality.py]: We want to keep these assertions

**Comment by cpurta** [comment]
Ahh must have looked over those with how Atom shows merge conflicts. Added those back.

**Comment by xanderdunn** [comment]
Great, thanks!  I need to figure out why CircleCI isn't running on this and then we'll be able to merge.

**Comment by xanderdunn** [comment]
@cpurta I had to revert this.  It has some serious problems:
```
find . -iname "*.py" ! -name "setup.py" ! -name "__init__.py" ! -path "./venv/*" | xargs pylint --rcfile=./.pylintrc
./submission_criteria/s3_util.py:57:26: F821 undefined name 'tempfile'
./submission_criteria/s3_util.py:59:30: F821 undefined name 'pd'
./submission_criteria/s3_util.py:60:63: E231 missing whitespace after ','
./submission_criteria/s3_util.py:60:71: E231 missing whitespace after ','
./submission_criteria/originality.py:55:32: E231 missing whitespace after ','
```

**Pull request merged**